web/src/flow/stages/authenticator_email/AuthenticatorEmailStage.ts

```diff
@@ -0,0 +1,165 @@
+            </header>
+            <div class="pf-c-login__main-body">
+                <form
+                    class="pf-c-form"
+                    @submit=${(e: Event) => {
+                        this.submitForm(e);
+                    }}
+                >
+                    <ak-form-static
+                        class="pf-c-form__group"
+                        userAvatar="${this.challenge.pendingUserAvatar}"
+                        user=${this.challenge.pendingUser}
+                    >
+                        <div slot="link">
+                            <a href="${ifDefined(this.challenge.flowInfo?.cancelUrl)}"
+                                >${msg("Not you?")}</a
+                            >
+                        </div>
+                    </ak-form-static>
+                    <ak-form-element
+                        label="${msg("Configure your email")}"
+                        required
+                        class="pf-c-form__group"
+                        .errors=${(this.challenge?.responseErrors || {})["email"]}
+                    >
+                        <input
+                            type="email"
+                            name="email"
+                            placeholder="${msg("Please enter your email address.")}"
+                            autofocus=""
+                            autocomplete="email"
+                            class="pf-c-form-control"
+                            required
+                        />
+                    </ak-form-element>
+                    ${this.renderNonFieldErrors()}
+                    <div class="pf-c-form__group pf-m-action">
+                        <button type="submit" class="pf-c-button pf-m-primary pf-m-block">
+                            ${msg("Continue")}
+                        </button>
+                    </div>
+                </form>
+            </div>
+            <footer class="pf-c-login__main-footer">
+                <ul class="pf-c-login__main-footer-links"></ul>
+            </footer>`;
+    }
+
+    renderEmailOTPInput(): TemplateResult {
+        return html`<header class="pf-c-login__main-header">
+                <h1 class="pf-c-title pf-m-3xl">${this.challenge.flowInfo?.title}</h1>
+            </header>
+            <div class="pf-c-login__main-body">
+                <ak-form-static
+                    class="pf-c-form__group"
+                    userAvatar="${this.challenge.pendingUserAvatar}"
+                    user=${this.challenge.pendingUser}
+                >
+                    <div slot="link">
+                        <a href="${ifDefined(this.challenge.flowInfo?.cancelUrl)}"
+                            >${msg("Not you?")}</a
+                        >
+                    </div>
+                </ak-form-static>
+                A verification token has been sent to your configured email address
+                ${this.challenge.email}
```

Comment:
```suggestion
                ${ifDefined(this.challenge.email)}
```

This may be null from the API, but should never happen in this case. Let's handle it anyway